Reload serve:test on test/index.html changes.

weltonrodrigo opened the pull request:
Sometimes one needs to include js files on index.html
Manually restarting serve:test means browersync will
open another tab on chrome and one needs to manually
reload all browsers.

Unless you do a quick fake change in a spec file to force reload.

As this is at most a very small improvement, feel free to discard
it as not wanted. ;D

sindresorhus commented on app/templates/gulpfile.js:
```diff
@@ -170,6 +170,7 @@ gulp.task('serve:test', () => {
   gulp.watch('app/scripts/**/*.js', ['scripts']);
 <% } -%>
+  gulp.watch('test/index.html').on('change', reload);
   gulp.watch('test/spec/**/*.js').on('change', reload);
```
`gulp.watch()` can accept an array, so just include it here. No need for multiple listeners.

weltonrodrigo replied:
I'll change and squash them.

weltonrodrigo committed:
Reload gulp:serve on test/index.html changes

Sometimes is needed to include a js file in test/index.html and it would be nice if browsers reload without manually restarting serve:test gulp task.

sindresorhus commented:
Thanks :)
